sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -0,0 +1,179 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.web.application.events;
+
+import static org.sormas.e2etests.pages.application.events.EditEventPage.*;
+
+import com.google.common.truth.Truth;
+import cucumber.api.java8.En;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import javax.inject.Inject;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.Event;
+
+public class EditEventSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  public static Event event;
+
+  @Inject
+  public EditEventSteps(WebDriverHelpers webDriverHelpers) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "^I change the event status to ([^\"]*)",
+        (String eventStatus) -> {
+          selectEventStatus(eventStatus);
+          webDriverHelpers.scrollToElement(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_DATA_SAVED_MESSAGE);
+        });
+    When("I collect the UUID displayed on Edit event page", () -> event = collectEventUuid());
+    When(
+        "I check the created data is correctly displayed in event edit page",
+        () -> {
+          event = collectEventData();
+          Truth.assertThat(event.getUuid()).isEqualTo(CreateNewEventSteps.newEvent.getUuid());
+          Truth.assertThat(event.getReportDate())
+              .isEqualTo(CreateNewEventSteps.newEvent.getReportDate());
+          Truth.assertThat(event.getEventDate())
+              .isEqualTo(CreateNewEventSteps.newEvent.getEventDate());
+          Truth.assertThat(event.getEventStatus())
+              .isEqualTo(CreateNewEventSteps.newEvent.getEventStatus());
+          Truth.assertThat(event.getInvestigationStatus())
+              .isEqualTo(CreateNewEventSteps.newEvent.getInvestigationStatus());
+          Truth.assertThat(event.getEventManagementStatus())
+              .isEqualTo(CreateNewEventSteps.newEvent.getEventManagementStatus());
+          Truth.assertThat(event.getRiskLevel())
+              .isEqualTo(CreateNewEventSteps.newEvent.getRiskLevel());
+          Truth.assertThat(event.getDisease()).isEqualTo(CreateNewEventSteps.newEvent.getDisease());
+          Truth.assertThat(event.getTitle()).isEqualTo(CreateNewEventSteps.newEvent.getTitle());
+          Truth.assertThat(event.getSourceType())
+              .isEqualTo(CreateNewEventSteps.newEvent.getSourceType());
+          Truth.assertThat(event.getEventLocation())
+              .isEqualTo(CreateNewEventSteps.newEvent.getEventLocation());
```

Comment:
Here we should use SoftAsserts. Also, we will have another POJOs comparing approach :D instead of keeping a lot of code here, we can create a EventComparator class where to compare objects. Check this PR to see an example: 
https://github.com/hzi-braunschweig/SORMAS-Project/pull/5561/files